client/src/app/models/user.model.ts

```diff
@@ -1,8 +1,6 @@
 export interface User {
```

Comment:
don't modify the interface we might this later, you can just create multiple interfaces for the different forms of the user that we are returning from the backend.

Reply:
As per @NicholasNagy 's request: "We should probably not be exposing anything other than the first name, last name, the role and whether they are active" I believe that leaving sensitive information from a request could lead to security issues. This comment refers to future possibilities which isn't the goal of the PR, I would ask to once again remember that PR is meant for displaying the client information in a table, having it be re-useable for other components and include a search button